types/webidl-conversions/webidl-conversions-tests.ts

```diff
@@ -31,7 +31,7 @@ conversions.any(any); // $ExpectType any
 conversions.any(unknown); // $ExpectType unknown
 conversions.any(options); // $ExpectType Options
 conversions.any(RegExp); // $ExpectType RegExpConstructor
-conversions.any(Symbol.toStringTag); // $ExpectType symbol
+conversions.any(Symbol.toStringTag); // $ExpectType symbol || typeof toStringTag
```

Comment:
I would expect that the stringified type would be `typeof Symbol.toStringTag` and not just `typeof toStringTag`.

Reply:
It would be if the test node printer specified an `enclosingDeclaration` for the node to be generated.